scenarios/arcaflow/memory-hog/input.yaml

```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
+node_selector: {}
+vm_bytes: 10%
+vm_workers: 2
```

Comment:
Can we translate this to provide ability for the user to define percentage of memory available to fill or certain amount of memory ( MB/GB ) to consume or a formula for user to tweak vm_bytes +vm_workers combination to get to it. Thoughts?

Reply:
can be expressed either in m,g,k or %, updated the documentation and the example input.yaml